lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -108,7 +144,8 @@ export default class LockMonitor {
 
     const currentLockState: LockState = {
       currentValueTimeLock: undefined,
-      latestSavedLockInfo: undefined
+      latestSavedLockInfo: undefined,
+      status: LockStatus.None

```

Comment:
Consider initializing `this.currentLockState` with this so `currentLockState` and just be type `LockState` so you don't have to use the `!` operater in line 104 and 114.

Reply:
Good idea .. done.